src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -119,4 +119,12 @@ public override object Display
             get { throw ExceptionUtilities.Unreachable; }
         }
     }
+
+    internal partial class BoundDefaultOperator
+    {
+        public override object Display
+        {
+            get { return "default"; }
```

Comment:
>return "default" [](start = 18, length = 16)

Should be `this.Type ?? "default"` (probably with a cast to `object` on one side or the other).